doc/SURVEYOPS/mtl/index.rst

```diff
@@ -0,0 +1,25 @@
+===
+mtl
+===
+
+``mtl`` is the root directory for the Merged Target List ledgers. The MTL
+ledgers record the state of each DESI target as redshift information is
+acquired. There is a set of ledgers for each of phase 2 of Survey Validation
+(``sv2``) the One-Percent Survey (``sv3``) and the Main Survey (``main``).
+
+The ``*done*ecsv`` files in the ``mtl`` directory track which tiles have
```

Comment:
Are there meant to be other "done" files besides `(scnd-|)mtl-done-tiles\.ecsv`?  In other words, you could be a lot more specific than `*done*ecsv`.

Reply:
Agreed. I updated this to read:

```
The ``mtl-done-tiles.ecsv`` and ``scnd-mtl-done-tiles.ecsv`` files in the ``mtl`` directory track which tiles...
```